.github/workflows/ros_translation_node.yml

```
@@ -0,0 +1,45 @@
+name: ROS translation node tests
```

Given how actions works, the name we use the first time will stay forever, so we might as well capitalize the title.

```suggestion
name: ROS Translation Node Tests
```

---

Do we ever expect to test against multiple ROS versions? if so should we mark this as ROS Humble?

---

Yes we can run this against a matrix of ROS versions. I started off with Humble for now, but if you prefer I can add others as well.

---

Perhaps we should have decided which ROS versions we support so we can update CI everywhere. For this PR, whoever, perhaps it's best to introduce the matrix and add at least two versions of ROS.

---

I added Jazzy